sources/manticore-balancer/Dockerfile

```diff
@@ -1,10 +1,11 @@
 FROM manticoresearch/manticore:5.0.0
 
 RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections && apt --fix-broken install -y && \
-apt-get update && apt-get install -y nano curl unzip git php-cli php-mysqlnd php-curl supervisor && rm -rf /var/lib/apt/lists/* && \
+apt-get update && apt -y install software-properties-common && add-apt-repository -y ppa:ondrej/php && apt-get update && \
```

Comment:
It's better to use apt-get instead of apt in Dockerfile

Reply:
This one is not resolved too.

Reply:
JFYI, 'apt' is not recommended in dockerfiles as having non-stable interface. 'apt-get' is used there